frontend/src/components/Header/components/CuratorAPIKeyGenerator/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Dialog, Intent, MenuItem } from "@blueprintjs/core";
+import {
+  Button as SDSButton,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+} from "czifui";
+import { useCallback, useRef, useState } from "react";
+import {
+  APIKeyResponse,
+  generateCuratorAuthKey,
+  useCuratorAuthKeyID,
+} from "src/common/queries/curation";
+import Toast from "src/views/Collection/components/Toast";
+import { APIDisclaimerP, FullWidthCallout, StyledInputGroup } from "./style";
+
+export default function CuratorAPIKeyGenerator(): JSX.Element {
+  const { data: apiKeyID } = useCuratorAuthKeyID();
```

Comment:
what's the difference between `useCuratorAuthKeyID()` and `generateCuratorAuthKey()`?

At first glance it seems like they're doing the same thing?

Reply:
ID will only return the ID, no key

Reply:
Ooh got it thanks! So ID is a GET and the key one is a POST/PUT

Reply:
Yes!